cli/command_server_start.go

```diff
@@ -222,7 +222,29 @@ func (c *commandServerStart) run(ctx context.Context, rep repo.Repository) error
 	return errors.Wrap(srv.SetRepository(ctx, nil), "error setting active repository")
 }
 
-func initPrometheus(mux *http.ServeMux) error {
+func (c *commandServerStart) setupHandlers(srv *server.Server, m *mux.Router) {
+	if c.serverStartUI {
+		srv.SetupHTMLUIAPIHandlers(m)
+	}
+
+	if c.serverStartLegacyRepositoryAPI {
+		srv.SetupRepositoryAPIHandlers(m)
+	}
+
+	if c.serverStartControlAPI {
+		srv.SetupControlAPIHandlers(m)
+	}
+
+	if c.serverStartUI {
+		if c.serverStartHTMLPath != "" {
+			srv.ServeStaticFiles(m, http.Dir(c.serverStartHTMLPath))
+		} else {
+			srv.ServeStaticFiles(m, server.AssetFile())
+		}
+	}
```

Comment:
Any reason for not including this block in the first `if c.serverStartUI {` above?
Are there any implicit dependencies?

Reply:
no reason, done.